next-project-directory/src/component/input.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+import styles from "./input.module.scss";
+
+interface Props {
+  placeholder: string;
+  disabled: boolean;
+  error: { message: string };
```

Comment:
추후에 error에 대한 정보가 message 외에 더 필요할수도 있겠지만, 지금 당장 요구사항에서는 아래와 같이 작성해주는게 좋을 것 같네요~
옵셔널도 추가해서요!
```suggestion
  errorMessage?: string;
```